itemfactory.py

```diff
@@ -0,0 +1,25 @@
+from engine.worldmap import WorldMap
+from random import randint, choice
+
+def initial_item_populate(new_world):
```

Comment:
What do you think about turning the limit variables into function parameters? And you can give them default values if you'd like.